aks/scfcharts.tf

```diff
@@ -20,6 +20,36 @@ resource "helm_release" "uaa" {
   values = [
     "${file("${local.chart_values_file}")}"
   ]
+  # scf-config-values
+  set {
+    name = "env.DOMAIN"
+    value = "${var.cap_domain}"
+  }
+  set {
+    name = "env.UAA_HOST"
+    value = "uaa.${var.cap_domain}"
+  }
+  set_sensitive {
+    name = "secrets.CLUSTER_ADMIN_PASSWORD"
+    value = "${var.cluster_admin_password}"
+  }
+  set_sensitive {
+    name = "secrets.UAA_ADMIN_CLIENT_SECRET"
+    value = "${var.uaa_admin_client_secret}"
+  }
+  # stratos-metrics-values
+  set {
+    name = "kubernetes.apiEndpoint"
+    value = "${var.cap_domain}"
+  }
+  set {
+    name = "metrics.username"
+    value = "${var.metrics_username}"
+  }
+  set_sensitive {
+    name = "metrics.password"
+    value = "${var.metrics_password}"
+  }
```

Comment:
As I mentioned the metrics params are used *only* by the metrics chart, not by uaa, scf or even the console chart.

Reply:
Well, that means either converting `stratos-metrics-values.yaml` into a template, or moving the helm deploy into terraform instead of `deploy_metrics.sh`. :/

Reply:
Done.